### DISCUSSION TITLE
CPE才兩題

### DISCUSSION THREAD

**Discussion by TobywithDino** [1101-GS4719-C/Python-Community-GS4719-C#1657] [General]
寫到腦子燒壞
然後第四題寫到一半電腦還給我當掉
黑畫面後重開，水喔，第四題給我重寫
然後第二題英文太爛，還不懂題目在講啥
我就爛，可以去休學了

**Comment by Cerosop** [comment]
第二題在去年的某一次考試中也有出現欸
他就是要你把輸入當成x進位，然後如過當成x進位後的值可以被x-1整除的話那x就是輸出

這題就是用for迴圈去遍歷x從2到62都試試看能不能整除就可以了
我覺得看到前3題都不用想太多直接解就能過了!
加油!

**Comment by TobywithDino** [comment]
我沒看到要整除的條件
一直在看測資
想說為甚麼
數字      最低Base
6     =>  7
61   =>  8
62   =>  9
出來才發現原來是有條件
我眼殘+英文不好

**Comment by 1chooo** [comment]
我之前考 CPE 的時候電腦也是在搞，交了一題給我等了半小時後才送出，甚至是畫面完全死當完全不能用的那種，結果後來那題 `WA`，完全沒看到那時候錯的程式碼，直接全部重打⋯⋯

**Comment by v0ouo** [comment]
我第四題也寫到快吐，寫了45分鐘都沒寫出來，然後一開始一直CE，後來才發現是編譯器問題，然後那個環境也超爛，我第一次用到打左括號不會幫你補右括號的，比DEVC++還爛，總之考一次有過畢業門檻就好，應該不會再考下一次了。

**Comment by mrweixd** [comment]
他的for迴圈還壞掉，真的是無法接受。

**Comment by chuanmin0317** [comment]
感覺能不能使用自己習慣的環境真的差蠻多的，像我在寫的時候for迴圈不小心讓array取到out of range但他沒報錯，在terminal看起來像是無限迴圈卡住，而且CPE旁邊的輸出串還跟正確的一樣但送出後一直錯誤，害我找了很久才找到錯在哪

**Comment by lyykey** [comment]
這邊推薦一項考 CPE 必備技能，在做任何的交檔案前，記得 Ctrl+A 然後 Ctrl+C，因為我過去也是遇過這種分人為因素的考試狀況，所以自此記得，無論如何都記得把剛打好的東西先複製起來。

**Comment by woohsin** [comment]
每次都會忘記要報名CPE，昨天點進去發現這次報名已經截止了，然後下次考試的報名時間又很難查...到現在CPE才一題，希望我能順利畢業。
